### ISSUE TITLE
Question about the baseline "w/o Adapt"

### ISSUE THREAD

**Issue by lyhdet** [canqin001/PointDAN#19]
I ran the script "train_source.py", and get the fellowing result, which is much higher than the reported, Am I missing something？

  | m2s | m2ss | s2m | ss2m | ss2s | avg
-- | -- | -- | -- | -- | -- | --
reported | 42.5 | 22.3 | 23.5 | 34.2 | 46.9 | 34.9
train_source.py | 63.76 | 28.94 | 25.78 | 45.79 | 59.03 | 45.92

supervised:
  | scan_net | shape_net | model_net
-- | -- | -- | --
reported | 53.2 | 86.2 | 90.5
train_source.py | 65.29 | 93.5 | 96.38

**Comment by canqin001** [comment]
Hi,

Did you run the baseline code under the same environments? We suggest re-run the code multiple times and compute the average one.

**Comment by adricarda** [comment]
Hi! Could you please specify if you average the best results or take the accuracy of the last iteration? I am also trying to reproduce the baselines.

**Comment by canqin001** [comment]
Hi,

We report the best average results by the last iteration. The environment is PyTorch 1.0 with Python 3.6. Here is the detail of our implementation (for both PointDA and baselines). The result is not quite stable as the initialized model weight will infect the performance. Please make sure to repeat the experiment multiple times to compute the average result.

![image](https://user-images.githubusercontent.com/46471476/105510442-a5212b00-5c9c-11eb-8e58-c2c7b86e8ccd.png)

**Comment by lyhdet** [comment]
Hi,
   I re-download the dataset & re-clone the code, re-run the "train_source.py" for 6 times, the only modify to code is dataset path, and number of data loader works:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/19669215/105566945-dae01580-5d69-11eb-8ced-fb302df86ae0.png">

here is the result (choose the best performance):



  | m2s | m2ss | s2m | s2ss | ss2m | ss2s | average
-- | -- | -- | -- | -- | -- | -- | --
reported | 42.5 | 22.3 | 39.9 | 23.5 | 34.2 | 46.9 | 34.9
rerun_1 | 63.8 | 24.14 | 52.34 | 28.49 | 50 | 60.96 | 46.62166667
rerun_2 | 66.49 | 29.23 | 46.03 | 23.01 | 51.17 | 63.44 | 46.56166667
rerun_3 | 58.19 | 32 | 46.96 | 26 | 42.67 | 54.49 | 43.385
rerun_4 | 62.38 | 31.43 | 43.86 | 23.18 | 45.44 | 58.03 | 44.05333333
rerun_5 | 63.92 | 26.91 | 42.06 | 22.61 | 50 | 56.74 | 43.70666667
rerun_6 | 60.15 | 34.77 | 54.67 | 19.9 | 45.79 | 62.2 | 46.24666667
avg_of_rerun | 62.48833333 | 29.74666667 | 47.65333333 | 23.865 | 47.51166667 | 59.31 | 45.09583333


  | scan_net | shape_net | model_net
-- | -- | -- | --
reported | 53.2 | 86.2 | 90.5
rerun_1 | 65.74 | 93.22 | 95.33
rerun_2 | 66.37 | 92.74 | 96.38
rerun_3 | 63.99 | 92.5 | 96.14
rerun_4 | 64.67 | 92.86 | 96.5
rerun_5 | 65.35 | 92.66 | 95.79
rerun_6 | 64.84 | 93.02 | 94.98
average | 65.16 | 92.83333333 | 95.85333333

here is the result (choose the last epoch):


  | m2s | m2ss | s2m | s2ss | ss2m | ss2s | average
-- | -- | -- | -- | -- | -- | -- | --
reported | 42.5 | 22.3 | 39.9 | 23.5 | 34.2 | 46.9 | 34.9
rerun_1 | 49.48 | 15.32 | 45.09 | 13.4 | 44.74 | 46.79 | 35.80333333
rerun_2 | 52.85 | 14.53 | 45.41 | 14.25 | 47.31 | 50.28 | 37.43833333
rerun_3 | 52.53 | 15.43 | 43.57 | 15.55 | 37.73 | 35.55 | 33.39333333
rerun_4 | 60.03 | 15.26 | 45.09 | 14.75 | 43.11 | 41.89 | 36.68833333
rerun_5 | 56.26 | 14.25 | 39.14 | 13.45 | 45.79 | 45.63 | 35.75333333
rerun_6 | 50.44 | 13.28 | 50.23 | 14.87 | 40.19 | 49.4 | 36.40166667
avg_of_rerun | 53.59833333 | 14.67833333 | 44.755 | 14.37833333 | 43.145 | 44.92333333 | 35.91305556


  | scan_net | shape_net | model_net
-- | -- | -- | --
reported | 53.2 | 86.2 | 90.5
rerun_1 | 64.39 | 92.74 | 94.51
rerun_2 | 63.93 | 92.26 | 95.21
rerun_3 | 61.39 | 91.57 | 94.98
rerun_4 | 63.31 | 92.7 | 95.21
rerun_5 | 63.37 | 91.89 | 94.39
rerun_6 | 64.1 | 92.17 | 93.46
average | 63.415 | 92.22166667 | 94.62666667



BTW, I use the NVIDIA 2080ti, which may different to your environment

**Comment by canqin001** [comment]
I guess the gap may come from the environment and the instability of models. The reported results are average ones over three repeats with some disturbance of performance. And we has cleaned a tiny portion of noisy and wrong data (~1%) before releasing this dataset which might further result in this performance gain. This dataset still has some room to improve and we plan to propose PointDA V2 dataset in the near future.

**Comment by lyhdet** [comment]
Thank you for your reply, the final baseline performance is similar to the reported.

**Comment by canqin001** [comment]
Yup. The testing accuracy disturb largely during the training. The overfitting towards source domain will hurt the generalization ability on the target domain. This comes from the domain gap and the small dataset. We guess such phenomenon can be removed by building a larger and more balanced dataset.
